test/testcafe/framework/page-objects/BasePageObject.js

```diff
@@ -7,4 +7,12 @@ export default class BasePageObject {
   async navigateToPage() {
     await this.t.navigateTo(`http://localhost:3000${this.url}`);
   }
+
+  getFormTitle() {
+    return this.form ? this.form.getTitle() : null;
```

Comment:
BasePageObject doesn't have `this.form` and it's wired to see the usage.
I think we could initialize form in `BasePageObject.initialize` instead of its children. Although not all page has Form but still slightly better in term of readability.